Formula/haxe.rb

```diff
@@ -32,7 +32,8 @@ def install
       ENV["OPAMROOT"] = buildpath/"opamroot"
       ENV["OPAMYES"] = "1"
       system "opam", "init", "--no-setup"
-      system "opam", "install", "ocamlfind"
+      system "opam", "switch", "4.04.2"
```

Comment:
Actually they updated the build system for the dev (4.x) branch but it requires merging of ptmap changes into the upstream opam package. It's been a while since that request but it's still pending. In the meanwhile this is the only way to compile the current git version.